modules/nci/include/imanager.h

```diff
@@ -13,6 +13,7 @@
 #include "frameContainer.h"
 #include "popup.h"
 #include "imanager.h"
+#include "keybindingContainer.h"
```

Comment:
Is it used?

Reply:
No, this is my error, I asked for add a keybinding and finally we only quit the app when ESC is pressed.